### ISSUE TITLE
Using JWT with presentations

### ISSUE THREAD

**Issue by David-Chadwick** [w3c/vc-data-model#586]
Suppose an ecosystem is using JWTs to sign VCs. Suppose after the holder has collected a set of JWT signed VCs, the holder wants to send a signed VP to a verifier that contains the signed VCs. It is not clear to me that our existing data model allows this. If you look at Section 4.10 and Example 12 which shows a VP encapsulating VCs, it is not clear:
a) if the encapsulated verifiableCredential allows a JWT VC to be encapsulated
b) if the VP can be signed using JWT as the text says "The contents of the proof property are proofs, as described by the Linked Data Proofs [LD-PROOFS] specification. "

**Comment by awoie** [comment]
@David-Chadwick According to the spec the proof attribute is optional.  A presentation containing a JWT VC would not contain a proof attribute. Instead it would use JWS as a proof as described in the example "EXAMPLE 31: Verifiable presentation using JWT compact serialization (non-normative)". Further, the verifiableCredential property could contain an array of "or of data derived from verifiable credentials in a cryptographically verifiable format.". IMO, a JWT represents this kind of verifiable format. For this reason, I believe it would be allowed according to the spec.

**Comment by David-Chadwick** [comment]
@awoie Thanks this clarifies things. Can I suggest we add the following sentence to the end of the text under example 12.

```
 An example of a
<a>verifiable presentation</a> using the JWT proof mechanism is given in section
<a href="#json web token"></a>.
```

**Comment by awoie** [comment]
@David-Chadwick I'm fine with that.

**Comment by David-Chadwick** [comment]
Added in PR #627

**Comment by msporny** [comment]
The PR has been merged, issue submitter agreed to change. Suggest we close.

**Comment by David-Chadwick** [comment]
Agreed

**Comment by burnburn** [comment]
WG resolution on 28 May 2019 call: 

RESOLUTION: Issue #586 is addressed via PR #627, which adds non-normative guidance wrt. using JWTs w/ presentations. Issue #586 should be closed since the PR has been merged.
